tslint.json

```diff
@@ -133,17 +133,28 @@
     }],
     "validate-decorators": [true, {
       "Component": {
-        "!host": "\\[class\\]",
-        "!preserveWhitespaces": ".*",
-        "!styles": ".*",
-        "!moduleId": ".*",
-        "changeDetection": "\\.OnPush$",
-        "encapsulation": "\\.None$"
+        "argument": 0,
+        "properties": {
+          "!host": "\\[class\\]",
+          "!preserveWhitespaces": ".*",
+          "!styles": ".*",
+          "!moduleId": ".*",
+          "changeDetection": "\\.OnPush$",
+          "encapsulation": "\\.None$"
+        }
       },
       "Directive": {
-        "!host": "\\[class\\]"
+        "argument": 0,
+        "properties": {
+          "!host": "\\[class\\]"
+        }
       },
-      "NgModule": "^(?!\\s*$).+"
+      "NgModule": {
+        "argument": 0,
+        "properties": {
+          "*": "^(?!\\s*$).+"
```

Comment:
It slightly feels confusing that `*` is part of `properties` but matches the actual decorator argument text (where it doesn't necessarily need to be a `ts.ObjectLiteralExpression`.

I understand why it is done that way, but it feels weird. Since it's just a lint rule though, it should be fine.

Reply:
I felt weird about it as well, but I wasn't sure how else to express it.

Reply:
We could have another property like `argumentText`, but I don't think it's too important.